package.json

```diff
@@ -10,7 +10,7 @@
   "license": "ISC",
   "dependencies": {
     "ethers": "^4.0.33",
-    "ganache-core": "^2.6.1",
```

Comment:
Wouldn't just  work to put npm ci, instead of npm install!

Reply:
I removed the hats in another branch

Reply:
I fixed this already with my PR